templates/base.html

```diff
@@ -87,29 +87,41 @@
 
     <a href="{{ url('home') }}" class="logo">{{ _('Mozilla Developer Network') }}</a>
 
-    <div class="header-login">
+    <div id="nav-sec">
         {% if not is_sphinx %}
-            {% include "includes/login.html" %}
+
+            {% if waffle.flag('header_tools') %}
+                  <ul>
+                  {% if user.is_authenticated() %}
+                      <li class="nav-tools"><a href="{{ wiki_url('MDN/Doc_status') }}">{{ _('Tools') }}<i aria-hidden="true" class="icon-caret-down"></i></a>
+
+                      <div class="submenu submenu-cols-2 js-submenu" id="nav-tools-submenu">
+                        <div class="submenu-column">
+                          <div class="title">{{ _('Contribute to MDN') }}</div>
+                          <ul>
+                            <li><a href="{{ wiki_url('MDN/Getting_started') }}">{{ _('Get started') }}</a></li>
+                            <li><a href="{{ wiki_url('MDN/Doc_status') }}">{{ _('Review or localize docs') }}</a></li>
+                            <li><a href="{{ wiki_url('Inbox') }}">{{ _('Add a new page') }}</a></li>
+                          </ul>
+                        </div><div class="submenu-column last">
+                          <div class="title">{{ _('Site Tools') }}</div>
+                          <ul>
+                            <li><a href="{{ url('dashboards.revisions') }}">{{ _('Revision dashboard') }}</a></li>
+                            <li><a href="{{ wiki_url('MDN/Doc_status') }}">{{ _('Documentation status') }}</a></li>
```

Comment:
Small concern that driving more traffic to this page could increase CPU load because of all the KumaScript that happens on the doc status pages. Let's just be sure to watch it closely in NR when we activate this waffle flag.